### PR TITLE
VACMS-2991 Add daily periodic task to reindex taxonomy entity index.

### PR DESCRIPTION
**Pull request by swirtSJW**
## Description

This should be enough to do it.... BUT running the drush command locally throws an error. SO do not merge this. 

Blocked by https://github.com/department-of-veterans-affairs/va.gov-cms/issues/3868
See #2991 . 

## QA steps

**Locally** 
- [ ] Confirm running `lando composer yaml-tests --tests-file=tasks-periodic.yml daily/update` in the terminal runs without errors.

**CI**
SSh into devshop environment
- [ ] Confirm running `composer yaml-tests --tests-file=tasks-periodic.yml daily/update` in the terminal runs without errors.

## Definition of Done
- [ ] Product release are written (or in progress), if required.
- [ ] Documentation has been updated, if applicable.

### CODE REVIEW

**Comment by swirtSJW** [comment]
This is the problem 
```
lando drush taxonomy-entity-index-rebuild

InvalidArgumentException: Route required in /app/docroot/core/lib/Drupal/Core/Url.php:167                                                                                                                                                            [error]
Stack trace:
#0 /app/docroot/core/includes/form.inc(880): Drupal\Core\Url::fromRouteMatch(Object(Drupal\Core\Routing\CurrentRouteMatch))
#1 /app/docroot/modules/contrib/taxonomy_entity_index/taxonomy_entity_index.drush.inc(59): batch_process()
#2 /app/docroot/vendor/drush/drush/includes/command.inc(422): drush_taxonomy_entity_index_rebuild()
#3 /app/docroot/vendor/drush/drush/includes/command.inc(231): _drush_invoke_hooks(Array, Array)
#4 /app/docroot/vendor/drush/drush/includes/command.inc(199): drush_command()
#5 /app/docroot/vendor/drush/drush/lib/Drush/Boot/BaseBoot.php(67): drush_dispatch(Array)
#6 /app/docroot/vendor/drush/drush/includes/preflight.inc(67): Drush\Boot\BaseBoot->bootstrap_and_dispatch()
#7 /app/docroot/vendor/drush/drush/drush.php(12): drush_main()
#8 {main}
[swirt@swirt-kudu] ~/workspace/va/vagov e7f07e34| VACMS-2991-periodically-rebuild-taxonomy-entity-index ✓ 
```

**Comment by swirtSJW** [comment]
This is on hold until this issue has been resolved https://www.drupal.org/project/taxonomy_entity_index/issues/3189721

**Comment by swirtSJW** [comment]
Works in Drush 10  .... fails in our Drush 8.

**Comment by swirtSJW** [comment]
This is blocked by https://github.com/department-of-veterans-affairs/va.gov-cms/issues/3868

**Comment by swirtSJW** [comment]
This one has been waiting for a looooong time.   With Drush 10 now out.... this should be ready to implement.

File: tasks-periodic.yml [hunk 1]
```diff
@@ -38,6 +38,11 @@ va/background/daily/migrate/nca_facility:
     - drush $DRUSH_ALIAS migrate:reset-status va_node_facility_nca
     - drush $DRUSH_ALIAS migrate:import va_node_facility_nca
 
+va/background/daily/update/taxonomy_entity_index:
+  description: Update the taxonomy_entity_index nightly.
+  command:
+    - drush $DRUSH_ALIAS taxonomy-entity-index-rebuild
```
Comment (indytechcook) [inline tasks-periodic.yml:44]:
```suggestion
    - drush $DRUSH_ALIAS taxonomy_entity_index:rebuild
```

Comment (indytechcook) [reply tasks-periodic.yml:44]: To match the drursh 10 default command

**Review by indytechcook — DISMISSED** [review]

**Comment by indytechcook** [comment]
I ran this on tugboat.  It did take a while to run which is ok on the daily since it runs overnight.

```
>  [notice] Message: 45038 results processed.
> 
 [notice] Batch operations end.
root@php:/var/lib/tugboat# 
```

**Comment by indytechcook** [comment]
@swirtSJW let's get this merged!

**Pull request merged**